libraries/Microsoft.Bot.Builder/Streaming/StreamingRequestHandler.cs

```diff
@@ -403,28 +403,24 @@ public async Task<ResourceResponse> SendActivityAsync(Activity activity, Cancell
                 }
             }
 
-            try
+            if (!_serverIsConnected)
```

Comment:
Thank you for removing that try-catch; the old behavior around this `if (!_serverIsConnected)` check was confusing.